packages/web/src/scss/components/Modal/index.html

```diff
@@ -1,150 +1,244 @@
 {{#> layout/plain }}
 
-  <button
-    type="button"
-    class="Button Button--primary Button--medium"
-    data-toggle="modal"
-    data-target="#modal-example-1"
-    aria-controls="modal-example-1"
-    aria-expanded="false"
-  >
-    Open Modal
-  </button>
-
-  <button
-    type="button"
-    class="Button Button--primary Button--medium"
-    data-toggle="modal"
-    data-target="#modal-example-2"
-    aria-controls="modal-example-2"
-    aria-expanded="false"
-  >
-    Open Modal with Long Content
-  </button>
-
-  <dialog id="modal-example-1" class="Modal">
-    <div class="Modal__content">
-      <div class="Modal__dialog">
-        <div class="Modal__header">
-          <button
-            type="button"
-            class="Button Button--tertiary Button--square Button--medium"
-            data-dismiss="modal"
-            data-target="#modal-example-1"
-            aria-controls="modal-example-1"
-            aria-expanded="false"
-          >
-            <svg class="Icon" width="24" height="24" aria-hidden="true">
-              <use xlink:href="/icons/svg/sprite.svg#close" />
-            </svg>
-            <span class="accessibility-hidden">Close</span>
-          </button>
-        </div>
-        <div class="Modal__body">
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquam at excepturi laudantium magnam mollitia
-            perferendis reprehenderit, voluptate. Cum delectus dicta ducimus eligendi excepturi natus perferendis
-            provident unde. Eveniet, iste, molestiae?
-          </p>
-        </div>
-        <div class="Modal__footer">
-          <button
-            type="button"
-            class="Button Button--primary Button--medium"
-          >
-            Button primary
-          </button>
-          <button
-            type="button"
-            class="Button Button--secondary Button--medium"
-            data-dismiss="modal"
-            data-target="#modal-example-1"
-          >
-            Close
-          </button>
+  <section class="docs-Section">
+
+    <h2 class="docs-Heading">Modal</h2>
+
+    <button
+      type="button"
+      class="Button Button--primary Button--medium"
+      data-toggle="modal"
+      data-target="#example_1"
+      aria-controls="example_1"
+      aria-expanded="false"
+    >
+      Open Modal
+    </button>
+
+    <!-- Modal: start -->
+    <dialog id="example_1" class="Modal Modal--composed" aria-labelledby="example_1_title">
+      <div class="Modal__content">
+        <div class="Modal__dialog">
+
+          <!-- Modal Header: start -->
+          <div class="ModalHeader">
+            <h2 id="example_1_title" class="ModalHeader__title">Modal Title</h2>
+            <button
```

Comment:
I know that we have a discussion about the `ModalButton` component. Should we have a ModalButton or CloseButton component that is part of the ModalHeader or ModalDialog? Eg. https://reactstrap.github.io/?path=/docs/components-modal--custom-close-button and https://github.com/reactstrap/reactstrap/blob/master/src/ModalHeader.js#L47

There was also a suggestion that the close button maybe should not be part of ModalHeader but ModalDialog.

Reply:
The same component is also in Tooltip. I think we should make this a separate component.

Reply:
I'm taking a note and we'll get back to it soon. Since we want to be in sync with designers, the decision cannot be just ours 🙂.